admin/app/dfp/CustomTargetingKeyAgent.scala

```diff
@@ -14,3 +14,15 @@ object CustomTargetingKeyAgent extends DataAgent[Long, String] {
     maybeData getOrElse Map.empty
   }
 }
+
+object CustomTargetingKeyService {
+
+  def targetingKey(session: SessionWrapper)(keyId: Long): String = {
+    lazy val fallback = {
+      val stmtBuilder = new StatementBuilder().where("id = :id").withBindVariableValue("id", keyId)
+      session.customTargetingKeys(stmtBuilder).head.getName
```

Comment:
Seems valid. I just moved this code from the location below. I'll add a get or else.